src/groundwater/aquifer.jl

```diff
@@ -111,12 +111,15 @@ instead. Specific yield will vary roughly between 0.05 (clay) and 0.45 (peat)
 """
 @get_units struct UnconfinedAquifer{T} <: Aquifer
     head::Vector{T} | "m"  # hydraulic head [m]
-    k::Vector{T} | "m d-1"  # horizontal conductivity [m d⁻¹]
+    kh₀::Vector{T} | "m d-1"  # reference horizontal conductivity [m d⁻¹]
```

Comment:
Nice to see this being implemented!
You could also store only `kh₀`, with the default option (uniform with depth) and the optional exponential decline? In other words, is there a need to store the actual horizontal conductivity `k`?

Reply:
Indeed, decided to only store `kh_0` (renamed back to `k` for consistency with the ConfinedAquifer), and the actual horizontal conductivity is removed from the struct.